src/lib/assets/yaml/overridesFile.ts

```diff
@@ -14,6 +14,33 @@ export type ChartOverrides = {
   name: string;
   image: string;
 };
+export type Probes = {
+  readinessProbe: {
+    httpGet: { path: string; port: number; scheme: "HTTPS" };
+    initialDelaySeconds: number;
+  };
+  livenessProbe: {
+    httpGet: { path: string; port: number; scheme: "HTTPS" };
+    initialDelaySeconds: number;
+  };
+};
+export type Resources = {
+  requests: { memory: string; cpu: string };
+  limits: { memory: string; cpu: string };
+};
+export type PodSecurityContext = {
+  runAsUser: number;
+  runAsGroup: number;
+  runAsNonRoot: true;
+  fsGroup: number;
+};
+export type ContainerSecurityContext = {
+  runAsUser: number;
+  runAsGroup: number;
+  runAsNonRoot: true;
+  allowPrivilegeEscalation: false;
+  capabilities: { drop: ["ALL"] };
+};
```

Comment:
```suggestion
type Probes = {
  readinessProbe: {
    httpGet: { path: string; port: number; scheme: "HTTPS" };
    initialDelaySeconds: number;
  };
  livenessProbe: {
    httpGet: { path: string; port: number; scheme: "HTTPS" };
    initialDelaySeconds: number;
  };
};
type Resources = {
  requests: { memory: string; cpu: string };
  limits: { memory: string; cpu: string };
};
type PodSecurityContext = {
  runAsUser: number;
  runAsGroup: number;
  runAsNonRoot: true;
  fsGroup: number;
};
type ContainerSecurityContext = {
  runAsUser: number;
  runAsGroup: number;
  runAsNonRoot: true;
  allowPrivilegeEscalation: false;
  capabilities: { drop: ["ALL"] };
};
```

I'm not certain these types need to be exported. If they do, this project has a `types.ts` file in `src/lib/` that traditionally holds exported types (not that that's necessarily the best way to do it?)

Reply:
This should be addressed now. Thanks so much for the feedback, I simplified a lot by not exporting these and then just hitting some additional edge cases on the tests from the top level function.